src/python/ensembl/io/genomio/gff3/extract_annotation.py

```diff
@@ -78,27 +78,26 @@ def __init__(self, provider_name: str = "") -> None:
     def get_xrefs(self, feature: SeqFeature) -> List[Dict[str, Any]]:
         """Get the xrefs from the Dbxref field."""
         all_xref: List[Dict[str, str]] = []
-        if not "Dbxref" in feature.qualifiers:
-            return all_xref
 
-        # Extract the Dbxrefs
-        for xref in feature.qualifiers["Dbxref"]:
-            dbname, name = xref.split(":", maxsplit=1)
-            if dbname == "GenBank" and self.provider_name == "RefSeq":
-                dbname = "RefSeq"
+        if "Dbxref" in feature.qualifiers:
+            for xref in feature.qualifiers["Dbxref"]:
+                dbname, name = xref.split(":", maxsplit=1)
+                if dbname == "GenBank" and self.provider_name == "RefSeq":
+                    dbname = "RefSeq"
 
-            if dbname.lower() in self.ignored_xrefs:
-                continue
+                if dbname.lower() in self.ignored_xrefs:
+                    continue
 
-            xrefs = {"dbname": dbname, "id": name}
-            all_xref.append(xrefs)
+                xrefs = {"dbname": dbname, "id": name}
+                all_xref.append(xrefs)
 
         # Add RefSeq ID xref if it looks like one
         if self.provider_name == "RefSeq":
             if feature.type == "gene" and feature.id.startswith("LOC"):
                 xref_dbs = {x["dbname"] for x in all_xref}
                 if "RefSeq" not in xref_dbs:
```

Comment:
Does this condition mean we cannot have more than 1 RefSeq xref? 
eg in such cases
```python
{
     	"description": "uncharacterized LOC129721096, transcript variant X4",
        "id": "LOC129721096_t1",
        "object_type": "transcript",
        "xrefs": [
            {
             	"dbname": "GeneID",
                "id": "129721096"
            },
	    {
                "dbname": "RefSeq",
                "id": "XR_008727343.1"
     	    }
        ]
    },

```

Reply:
Yes, there should never be several RefSeq IDs for the same feature. In this example it's not a gene so it's not impacted; we store correctly the transcript RefSeq ID.

Reply:
ahh yes this is a transcript, makes sense. Thank you!!